### PR TITLE
Fix #9 Add React router dom

### LINKED ISSUE #9: Add React Router
Add React Router

### PR DESCRIPTION
**Pull request by rray524**
Fix #9 Add React router dom
### Description

This pull request adds detailed instructions on how to set up and use react-router-dom in a React project. This PR also includes the installation of the react-router-dom npm package and necessary updates to the project's configuration files.

### Type of Change

* Added react-router-dom npm package to the project.
* Configuration details for setting up routing in the project `src/main.jsx` file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
   },
   "dependencies": {
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "react-router-dom": "^6.26.0"
```
Comment (rt4914) [inline package.json:15]: Along with this do the basic setup of using `createBrowserRouter` in main.jsx

https://reactrouter.com/en/main/start/tutorial

**Review by rt4914 — CHANGES_REQUESTED** [review]
@rray524 Suggested change

File: package.json (hunk 1 above)
Comment (rray524) [reply package.json:15]: @rt4914 hi, added `createBrowserRouter` in main.jsx file

**Review by rt4914 — APPROVED** [review]
LGTM, Also please update your branch by merging with latest main after which we. can merge this PR.

**Comment by rray524** [comment]
> LGTM, Also please update your branch by merging with latest main after which we. can merge this PR.

@rt4914  updated branch by merging with latest main. Please check for merging it.

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**